libs/miner/include/miner/block_optimiser.hpp

```diff
@@ -1,457 +0,0 @@
-#pragma once
```

Comment:
Can we keep the annealer miner stuff. We have an open task to reinstate this

Reply:
The following files have been saved from certain doom: 
libs/miner/include/miner/annealer_miner.hpp
 libs/miner/include/miner/block_optimiser.hpp
 libs/miner/include/miner/optimisation/binary_annealer.hpp

Reply:
👍